ios/FluentUI/Tooltip/TooltipTokenSet.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Design token set for the `Tooltip` control.
+public class TooltipTokenSet: ControlTokenSet<TooltipTokenSet.Tokens> {
+    public enum Tokens: TokenSetKey {
+        /// The color of the body of the tooltip.
+        case tooltipColor
+
+        /// The color of the text within the tooltip.
+        case textColor
+
+        /// The margins from the window's safe area insets used for laying out the tooltip.
+        case shadowInfo
+
+        /// The radius for the corners of the tooltip.
+        case backgroundCornerRadius
+
+        /// The TextStyle of the message label.
+        case messageLabelTextStyle
+
+        /// The TextStyle of the title label.
+        case titleLabelTextStyle
+
+        /// The horizontal padding between the text and edges of the tooltip.
+        case paddingHorizontal
+
+        /// The vertical padding between the text and edges of the tooltip with both a title and message.
+        case paddingVerticalWithTitle
```

Comment:
similar to notfication tokens, i thought the spacings are not tokenizable. we should only expose public tokens that we are going to repsect. text color, style seems reasonable. maybe get clarification from design team?

Reply:
talked with Mike since he made the initial change, we're going to make them token constants!